templates/ViewHandler.inc.php

```diff
@@ -126,7 +126,7 @@ class="form-control form-control-sm"
                     $origlist[$num][$j - $k] = $item[$j - $k];
                     if (str_starts_with($row["type"], "lie")) {
                         if (empty($cached_options[$row["field"]])) {
-                            $options = (new Table($row["sql_table"], $row["sql_columns"]))->get_list($form->DBHandle, str_replace("%id", $item[$j - $k], $row["sql_clause"]));
```

Comment:
This looks like it was pulled from an outdated branch. Anyway it's fixed in commit 33875933